Sources/Orbit/Support/Accessibility/AccessibilityID.swift

```diff
@@ -1,57 +1,73 @@
-    case alertDescription               = "orbit.alert.description"
-    case cardTitle                      = "orbit.card.title"
-    case cardIcon                       = "orbit.card.icon"
-    case cardDescription                = "orbit.card.description"
-    case cardActionButtonLink           = "orbit.card.action.buttonLink"
-    case checkboxTitle                  = "orbit.checkbox.title"
-    case checkboxDescription            = "orbit.checkbox.description"
-    case choiceTileTitle                = "orbit.choicetile.title"
-    case choiceTileIcon                 = "orbit.choicetile.icon"
-    case choiceTileDescription          = "orbit.choicetile.description"
-    case choiceTileBadge                = "orbit.choicetile.badge"
-    case dialogTitle                    = "orbit.dialog.title"
-    case dialogDescription              = "orbit.dialog.description"
-    case dialogButtonPrimary            = "orbit.dialog.button.primary"
-    case dialogButtonSecondary          = "orbit.dialog.button.secondary"
-    case dialogButtonTertiary           = "orbit.dialog.button.tertiary"
-    case emptyStateTitle                = "orbit.emptystate.title"
-    case emptyStateDescription          = "orbit.emptystate.description"
-    case emptyStateButton               = "orbit.emptystate.button"
-    case fieldLabel                     = "orbit.field.label"
-    case fieldMessage                   = "orbit.field.message"
-    case fieldMessageIcon               = "orbit.field.message.icon"
-    case inputPrefix                    = "orbit.input.prefix"
-    case inputSuffix                    = "orbit.input.suffix"
-    case inputValue                     = "orbit.input.value"
-    case inputFieldPasswordToggle       = "orbit.inputfield.password.toggle"
-    case keyValueKey                    = "orbit.keyvalue.key"
-    case keyValueValue                  = "orbit.keyvalue.value"
-    case listChoiceTitle                = "orbit.listchoice.title"
-    case listChoiceIcon                 = "orbit.listchoice.icon"
-    case listChoiceDescription          = "orbit.listchoice.description"
-    case listChoiceValue                = "orbit.listchoice.value"
-    case passwordStrengthIndicator      = "orbit.passwordstrengthindicator"
-    case radioTitle                     = "orbit.radio.title"
-    case radioDescription               = "orbit.radio.description"
-    case tileTitle                      = "orbit.tile.title"
-    case tileIcon                       = "orbit.tile.icon"
-    case tileDescription                = "orbit.tile.description"
-    case tileDisclosureButtonLink       = "orbit.tile.disclosure.buttonlink"
-    case tileDisclosureIcon             = "orbit.tile.disclosure.icon"
+///
+/// Can be extended to provide custom accessibility identifiers.
+public struct AccessibilityID: RawRepresentable {
+
+    static let alertButtonPrimary           = Self(rawValue: "orbit.alert.button.primary")
```

Comment:
What about moving the acutal definitions to an extension? That way, it will be the same as in clients and the primary declaration can focus on `RawRepresentable` only.

Reply:
Good idea - and it can probably go as extension in each component file separately, right?

Reply:
Like so?

<img width="566" alt="image" src="https://user-images.githubusercontent.com/35844477/199484740-b3a87e2d-3d58-4def-968c-44b0ee6a36fc.png">

Reply:
Why not, although are there no cases where an identifier is used in multiple files?

Reply:
Did not happen so far. Even if yes, it would be defined in the component it primarily belongs to.